tests/test_tabular_exporter.py

```diff
@@ -159,6 +159,7 @@ def test_export_to_csv_response(self):
                                    '1,2\r\n', '3,4\r\n', 'abc,def\r\n',
                                    '2015-08-28T00:00:00,2015-08-28\r\n'])
 
+    @unittest.skip("failing - does not return an HttpResponse")
```

Comment:
I was not expecting that…

Reply:
yeah, I wasn't sure how to troubleshoot this - I skipped it for now so I could make sure the build was working

could it be something I don't have installed? or something that changed with debug behavior?

Reply:
Yeah, I'm waiting for the punchline to be that it's returning a redirect or something like that.

Reply:
there is a redirect but when I added the arg to follow it rendered an admin list template which I didn't think was the expected behavior (based on my memory from the other day)

Reply:
Ah, running it without the skip:

```
TypeError: never_cache didn't receive an HttpRequest. If you are decorating a classmethod, be sure to use @method_decorator.```

Reply:
Okay, so it looks like this was a backwards-incompatible change in Django but there's an easy fix to switch to `add_never_cache_headers` which is simpler anyway.